pkg/grpcutils/utils.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package grpcutils
+
+import (
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```

Comment:
The `grpc_middleware` was initially avoided because it's bringing in quite a lot of dependencies.

Reply:
oh, okay, i can take it out. the main thing i care about is that we keep stream and unary interceptors together. that's super important, regardless of if we use a chaining interceptor helper

Reply:
wait, what dependencies? it didn't add more than itself to go.sum. is that because we already depend on its dependencies?